app-header/app-header.html

```diff
@@ -95,6 +95,9 @@
       position: relative;
       display: block;
       -webkit-transform: translateZ(0);
+      transition-property: -webkit-transform;
+      transition-property: transform;
```

Comment:
The second one will overwrite the first one since both are setting on the same CSS property (transition-property).  If you need to support prefix one it needs to use the prefix transition, e.g.
```
-webkit-transition-property: -webkit-transform;
transition-property: transform;
```